pkg/operator/k8sutil/prometheus.go

```diff
@@ -0,0 +1,114 @@
+package k8sutil
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+
+	monitoringv1 "github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1"
+	monitoringclient "github.com/coreos/prometheus-operator/pkg/client/versioned"
+	"k8s.io/apimachinery/pkg/api/errors"
+	k8sYAML "k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+const (
+	monitoringPath           = "/etc/ceph-monitoring/"
+	prometheusRuleConfigJSON = "prometheus_rule_config.json"
+)
+
+func getMonitoringClient() (*monitoringclient.Clientset, error) {
+	cfg, err := clientcmd.BuildConfigFromFlags("", "")
+	client, err := monitoringclient.NewForConfig(cfg)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get monitoring client. %+v", err)
+	}
+	return client, nil
+}
+
+// GetServiceMonitor returns servicemonitor or an error
+func GetServiceMonitor(filePath string) (*monitoringv1.ServiceMonitor, error) {
+	file, err := ioutil.ReadFile(monitoringPath + filePath)
```

Comment:
when concatenating file paths it's preferred to use `path.Join(monitoringPath, filePath)`. There are several instances I see below where this change should also be made.

Reply:
ack.